index.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   minio-operator:
   - apiVersion: v2
-    appVersion: v4.2.0
+    appVersion: v4.2.1
```

Comment:
Don't edit this yourself @dvaldivia this is generated automatically by `helm-reindex.sh`

Reply:
was not aware, just ran it, it updated the digest and updated a tar, is it supposed to do that?

Reply:
Yes